apps/etterlatte-beregning/src/main/kotlin/ytelseMedGrunnlag/YtelseMedGrunnlagService.kt

```diff
@@ -34,7 +34,7 @@ class YtelseMedGrunnlagService(
                     avkorting.avkortingGrunnlag
                         .filter { it.fom <= avkortetYtelse.fom }
                         .maxBy { it.fom }
-                val aarsinntekt = grunnlag.aarsinntekt + grunnlag.inntektUtland
+                val aarsinntekt = grunnlag.inntektTom + grunnlag.inntektUtlandTom
```

Comment:
Vurdere å gjøre noe med denne også, og i DTO? ser for meg at det er litt mer stress med tanke på at det kan gjelde flere apper da :p

Reply:
Det er kun brev. Vi må fort gjøre noe med brevinnholdet også. Jeg endrer det da.